temporal_capi/src/plain_date.rs

```diff
@@ -155,6 +156,39 @@ pub mod ffi {
                 .map_err(Into::into)
         }
 
+        pub fn equals(&self, other: &Self) -> bool {
+            self.0 == other.0
+        }
+
+        pub fn compare(one: &Self, two: &Self) -> i32 {
+            Self::compare_iso_date(
+                one.iso_year(),
+                one.iso_month(),
+                one.iso_day(),
+                two.iso_year(),
+                two.iso_month(),
+                two.iso_day(),
+            )
+        }
+
+        pub fn compare_iso_date(
```

Comment:
Please use the library's compare function. Don't write your own. only write a thing wrapper.